package.json

```diff
@@ -5,11 +5,8 @@
   "main": "index.js",
   "type": "module",
   "scripts": {
-    "start": "node server.js",
-    "server": "nodemon server.js",
-    "client": "npm start --prefix client",
-    "dev": "concurrently \"npm run server\" \"npm run client\"",
-    "heroku-postbuild": "NPM_CONFIG_PRODUCTION=true"
+    "server": "node server.js",
+    "server-nodemon": "nodemon server.js"
```

Comment:
I think, it's better to not touch this one. It's perfectly working so let's leave it as is.

Overall, other changes looks good to me.